portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,12 +46,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       Date resultdate = new Date(timestamp);
       String date = sdf.format(resultdate);
       long postId = Long.parseLong(request.getParameter("id"));
+      String userEmail = null;
+
+      UserService userService = UserServiceFactory.getUserService();
+      if (userService.isUserLoggedIn()){
+        userEmail = userService.getCurrentUser().getEmail();
+      }else{
```

Comment:
Could you apply formatter please. (Spaces between if/else)

Reply:
done.